packages/Liquid-Core.package/LiquidAnswerPanel.class/instance/selectedChoices..st

```diff
@@ -0,0 +1,4 @@
+accessing
+selectedChoices: anObject
+
+	selectedChoices := anObject.
```

Comment:
better naming?

Reply:
Are you referring to the Method/instance-variable name or the "anObject"? If the former: do you have any suggestions? If the latter: "anObject" would be a Set, I have changed the name to better reflect that.